codegen-client/src/main/kotlin/software/amazon/smithy/rust/codegen/client/smithy/CodegenVisitor.kt

```diff
@@ -105,6 +103,8 @@ class CodegenVisitor(
      */
     internal fun baselineTransform(model: Model) =
         model
+            // Convert string shapes with an @enum trait to an enum shape
+            .let { ModelTransformer.create().changeStringEnumsToEnumShapes(model, true) }
```

Comment:
There is a test that ensures mixins are transformed properly by the line below. Perhaps we should add a test that verifies this transform is present as well.

Reply:
Added, thank you